src/Npgsql.FSharp.fs

```diff
@@ -227,6 +227,7 @@ module Sql =
             | SqlValue.Bool boolean -> add boolean NpgsqlDbType.Boolean
             | SqlValue.Decimal number -> add number NpgsqlDbType.Numeric
             | SqlValue.Money number -> add number NpgsqlDbType.Money
+            | SqlValue.Long number -> add number NpgsqlDbType.Smallint
```

Comment:
This should be `SqlValue.Short` instead of `SqlValue.Long`, right?

Reply:
Yes, the copy-paste has betrayed me 😿

Reply:
``SqlValue.Short``  was already in the match expression in line 222, so I will remove the line. Sounds good?

Reply:
That is even better, thanks!